cypress/lib/helper.js

```diff
@@ -35,4 +35,36 @@ export function checkProspectsAreVisible() {
   cy.contains(prospect1)
   cy.contains("Prospect 2")
   cy.contains("Prospect 10")
+}
+
+export function checkBookingsOrder(expectedData) {
+  for (let index = 0; index < expectedData.length; ++index) {
+    cy.get('p').eq(index).contains(expectedData[index])
+  }
+}
+
+export function createExpectedData(firstBooking, ordering, bookingsNumber) {
```

Comment:
why do we need this function? Shouldn't we create the data only once and sort it based on different criteria?

Reply:
wouldn't it be much harder to sort the data than to create new ones?

Reply:
no, if we create the data in such a way that each field that we order it by will yield a different ordering, it is much easier to understand than the function